install/020-ConfigMap.yaml

```diff
@@ -10,8 +10,8 @@ data:
     bridge.id=my-bridge
 
     #MQTT server common
-    mqtt.host=0.0.0.0
-    mqtt.port=1883
+    mqtt.server.host=0.0.0.0
+    mqtt.server.port=1883
```

Comment:
I made this change. Out of interest, since since you've tagged the PR for 0.2.0, shouldn't this have been in? Will the files here be updated by the action building the release?

Reply:
It will be set here after the release, by getting what's in there the packaging folder.